_unused_topics/cluster-logging-uninstall-cluster-ops.adoc

```diff
@@ -5,7 +5,7 @@
 [id="cluster-logging-uninstall-ops_{context}"]
 = Uninstall the infra cluster
 
-You can uninstall the infra cluster from the {product-title} cluster logging. 
+You can uninstall the infra cluster from the OpenShift Logging. 
```

Comment:
In this example, "the OpenShift Logging" sounds odd to me. Suggest dropping "the"